tools/cldr-apps/src/test/java/org/unicode/cldr/web/TestUserLevel.java

```diff
@@ -0,0 +1,241 @@
+            () -> assertFalse(UserRegistry.userIsExactlyManager(a_anonymous), a_anonymous + " is exactly manager"),
+
+            () -> assertTrue(UserRegistry.userCreateOtherOrgs(a_admin), a_admin + " can create orgs"),
+            () -> assertFalse(UserRegistry.userCreateOtherOrgs(a_tc), a_tc + "can create orgs"),
+            () -> assertFalse(UserRegistry.userCreateOtherOrgs(a_manager), a_manager + " can create orgs"),
+            () -> assertFalse(UserRegistry.userCreateOtherOrgs(a_vetter), a_vetter + " can create orgs"),
+            () -> assertFalse(UserRegistry.userCreateOtherOrgs(a_locked), a_locked + " can create orgs"),
+            () -> assertFalse(UserRegistry.userCreateOtherOrgs(a_anonymous), a_anonymous + " can create orgs"),
+
+            () -> assertTrue(UserRegistry.userCanModifyUsers(a_admin), a_admin + " can modify users"),
+            () -> assertTrue(UserRegistry.userCanModifyUsers(a_tc), a_tc + "can modify users"),
+            () -> assertTrue(UserRegistry.userCanModifyUsers(a_manager), a_manager + " can modify users"),
+            () -> assertFalse(UserRegistry.userCanModifyUsers(a_vetter), a_vetter + " can modify users"),
+            () -> assertFalse(UserRegistry.userCanModifyUsers(a_locked), a_locked + " can modify users"),
+            () -> assertFalse(UserRegistry.userCanModifyUsers(a_anonymous), a_anonymous + " can modify users"),
+
+            () -> assertTrue(UserRegistry.userCanEmailUsers(a_admin), a_admin + " can email users"),
+            () -> assertTrue(UserRegistry.userCanEmailUsers(a_tc), a_tc + "can email users"),
+            () -> assertTrue(UserRegistry.userCanEmailUsers(a_manager), a_manager + " can email users"),
+            () -> assertFalse(UserRegistry.userCanEmailUsers(a_vetter), a_vetter + " can email users"),
+            () -> assertFalse(UserRegistry.userCanEmailUsers(a_locked), a_locked + " can email users"),
+            () -> assertFalse(UserRegistry.userCanEmailUsers(a_anonymous), a_anonymous + " can email users"),
+
+            () -> assertTrue(UserRegistry.userCanEmailUsers(a_admin), a_admin + " can email users"),
+            () -> assertTrue(UserRegistry.userCanEmailUsers(a_tc), a_tc + "can email users"),
+            () -> assertTrue(UserRegistry.userCanEmailUsers(a_manager), a_manager + " can email users"),
+            () -> assertFalse(UserRegistry.userCanEmailUsers(a_vetter), a_vetter + " can email users"),
+            () -> assertFalse(UserRegistry.userCanEmailUsers(a_locked), a_locked + " can email users"),
+            () -> assertFalse(UserRegistry.userCanEmailUsers(a_anonymous), a_anonymous + " can email users"),
+
+
+            () -> assertTrue(true, "True is true!")
+        );
+        */
+    }
+
+    static int id = 9468; // See TestUserRegistry.java
+
+    static UserRegistry.User getUser(UserRegistry reg, Organization o, VoteResolver.Level l) {
```

Comment:
this looks like potentially reusable code for tests in general -- worth making it a separate class and file?

Reply:
or moving it into UserRegistry.  I think i might update it to cache, to see if there's already an `adlam_admin@example.org` registered and use that.